parent/pom.xml

```diff
@@ -546,8 +557,11 @@
           <artifactId>maven-compiler-plugin</artifactId>
           <version>${plugin.version.compiler}</version>
           <configuration>
-            <source>1.8</source>
-            <target>1.8</target>
+            <release>11</release>
+
+            <!-- WE NEED THIS FOR IDEA Intellij -->
+            <source>11</source>
+            <target>11</target>
```

Comment:
The compiler plugin is already configured In the parent pom `camunda-release-parent` like:

```
<plugin>
          <groupId>org.apache.maven.plugins</groupId>
          <artifactId>maven-compiler-plugin</artifactId>
          <version>${plugin.version.compiler}</version>
          <configuration>
            <source>${version.java}</source>
            <target>${version.java}</target>
          </configuration>
        </plugin>
```

So, we just need to change the property `version.java` and setting the release version.

Reply:
Please see latest commit